servers/cromwell/README.md

```diff
@@ -38,15 +38,33 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     {
       "field": "labels.cromwell-workflow-id",
       "display": "Workflow ID"
+    },
```

Comment:
Might not add this to the README quite yet since it won't make any visible changes to UI.